beacon-chain/powchain/service.go

```diff
@@ -525,3 +575,19 @@ func (w *Web3Service) requestBatchedLogs() error {
 	w.lastRequestedBlock.Set(requestedBlock)
 	return nil
 }
+
+// PrometheusToFloat64 convert counters to float64
+func PrometheusToFloat64(m prometheus.Metric) float64 {
```

Comment:
Should this be under `/powchain/service.go`? I think it should be part of `shared`

Reply:
should I reply with done or just resolve when I fix?